modules/precompiles/test/testHelpers.ts

```diff
@@ -0,0 +1,94 @@
+import { ethers } from "hardhat";
+import { Log, Interface } from "ethers";
+import { expect } from "chai";
+
+/**
+ * Asserts that a transaction reverts with a specific error message.
+ * @param tx The transaction promise that should revert.
+ * @param expectedError The expected revert error message.
+ */
+export const expectRevert = async (tx: Promise<any>, expectedError: string) => {
+    try {
+        await tx;
+        expect.fail(`Expected transaction to revert with '${expectedError}', but it did not`);
+    } catch (error: any) {
+        // console.log("Full error message:", error);
+        expect(error.message).to.contain(expectedError);
+    }
+};
+
+/**
+ * @param ownerContract The contract instance connected to the owner signer.
+ * @param userContract The contract instance connected to the user signer.
+ * @param ownerSigner The owner signer object.
+ * @param userSigner The user signer object.
+ */
+export async function resetOwnerState(
```

Comment:
Respecte a si utilitzar funcions o arrow functions per a les funcions, no tenim cap preferencia pero si que esta be que nomes siguin d'un tipus a no ser que sigui necessari